Sources/Profile-View.php

```diff
@@ -699,7 +699,7 @@ function showPosts($memID)
 		}
 
 		// Make sure we quit this loop.
-		if ($smcFunc['db_num_rows']($request) === $maxIndex || $looped)
+		if ($smcFunc['db_num_rows']($request) === $maxIndex || $looped || $range_limit == '')
```

Comment:
Am I missing something? The body of this loop seems to ever run 1 time. Why is this `while` loop in place?

Reply:
The first run try something,
when it not successed it try to do the same again but with range_limit  = ''.
Dunno how i get to the state that range_limit is in the first run not empty.